net/DevExtreme.AspNet.Data/DataSourceLoadOptionsBase.cs

```diff
@@ -67,12 +67,21 @@ public class DataSourceLoadOptionsBase {
         /// </summary>
         public string[] Select { get; set; }
 
+        /// <summary>
+        /// An array of data fields that constrains the <see cref="Select" /> expression.
+        /// The applied select expression is the intersection of <see cref="PreSelect" /> and <see cref="Select" />.
```

Comment:
applied select expression? Perhaps you should indicate the 'select' is the name of the expression?
"is the intersection of" Explain.